ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.23__47.yaml

```diff
@@ -41,6 +41,14 @@ images:
       - destination_dir: .
         source_path: /go/bin/distributed_dispatcher
   to: knative-eventing-kafka-distributed-dispatcher
+- dockerfile_path: openshift/ci-operator/knative-images/migrate/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/migrate
```

Comment:
error: error copying source path /go/bin/migrate to .: exit status 1

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/19881/rehearse-19881-pull-ci-openshift-knative-eventing-kafka-release-v0.23-48-images/1410558764695687168

Reply:
yes, depends on the other PR